src/mumble/AudioInput.cpp

```diff
@@ -548,6 +548,30 @@ void AudioInput::initializeMixer() {
 }
 
 void AudioInput::addMic(const void *data, unsigned int nsamp) {
+	// check if there are trailing zeros in the sample and adjust nsamp
+	if (nsamp > 0) {
+		if (eMicFormat == SampleFloat) {
+			const auto *float_data = reinterpret_cast< const float * >(data);
```

Comment:
Why use `auto` instead of `float` here? Doesn't seem to save you any typing :thinking: 

Personally I am not a fan of using auto :shrug:

Reply:
I don't use auto a lot... But the clang-tidy linter recommended me to do so. Do you think I should change them back?

Reply:
It probably did so because of the explicit cast on the right side of the assignment...
Personally I would not use auto here, but I also don't have a strong opinion on that in this case, so if you want to you can also leave it as it is. Your call :)

Reply:
Okay... I have already removed auto. I think I can keep auto for more complicated and tedious tasks.